public/app/plugins/panel/geomap/dims/editors/ColorDimensionEditor.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC, useCallback } from 'react';
+import { GrafanaTheme2, SelectableValue, StandardEditorProps } from '@grafana/data';
+import { ColorDimensionConfig } from '../types';
+import { Select, ColorPicker, useStyles2 } from '@grafana/ui';
+import {
+  useFieldDisplayNames,
+  useSelectOptions,
+} from '../../../../../../../packages/grafana-ui/src/components/MatchersUI/utils';
```

Comment:
typically this is bad... but I do not yet want to export them from `@grafana/ui`